v1/examples/pipeline/stages/approval/approval-stage.yaml

```diff
@@ -6,7 +6,7 @@ spec:
       name: approval
       id: Approval
       timeout: 10m
-      when: <+environment.name? != "QA"
+      when: <+environment.name> != "QA"
```

Comment:
change this to `<+<+environment.name> != "QA">`

Reply:
Done